### PR TITLE
Issue-33 Fixing comments from issue conversation 

### PR DESCRIPTION
**Pull request by dtorresf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,18 +12,26 @@ This module creates an AWS CloudTrail trail and the necessary AWS resources to s
...
 module "cloudtrail" {
-  source                       = "github.com/lincolnloop/terrraform-aws-cloudtrail.git"
+  source                       = "github.com/lincolnloop/terraform-aws-cloudtrail.git"
   name                         = "cloudtrail"
-  description                  = "Cloudtrail logs"
+  description                  = "Cloudtrail"
+  cloudtrail_config            = var.cloudtrail_config
+  cloudwatch_log_group_name    = "/cloudtrail"
+  cloudwatch_retention_in_days = 1
+  cloudtrail_iam_role_name     = "CloudWatchRokes"
```
Comment (ipmb) [inline README.md]: typo here?

File: README.md [hunk 2]
```diff
@@ -12,18 +12,26 @@ This module creates an AWS CloudTrail trail and the necessary AWS resources to s
...
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| cloudtrail_config | Configuration for CloudTrail alarms and tags. | `object` | Review next section for default value | yes |
+| alarm_action_arns | List of ARNs for the alarm actions (e.g., SNS topic ARNs) | `list(string)` | no | yes |
+| cloudtrail_tags | Configuration for CloudTrail alarms and tags. | `object` | Review next section for default value | yes |
+| cloudtrail_alarms | Configuration for CloudTrail alarms and tags. | `object` | Review next section for default value | yes |
 | cloudtrail_iam_role_name | IAM Role name for CloudTrail Role. | `string` | `CloudTrail` | no |
 | cloudtrail_sns_topic_name | SNS topic name for cloudtrail. | `string` | `aws-cloudtrail` | no |
 | cloudwatch_log_group_name | Name for the CloudWatch log group that will contain the cloudtrail logs. | `string` | `/cloudtrail` | no |
```
Comment (ipmb) [inline README.md]: Can this default to `/aws/cloudtrail` to match their typical naming scheme?

File: README.md [hunk 3]
```diff
@@ -12,18 +12,26 @@ This module creates an AWS CloudTrail trail and the necessary AWS resources to s
...
+
 ```
 
 ## Inputs
 
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
-| cloudtrail_config | Configuration for CloudTrail alarms and tags. | `object` | Review next section for default value | yes |
+| alarm_action_arns | List of ARNs for the alarm actions (e.g., SNS topic ARNs) | `list(string)` | no | yes |
+| cloudtrail_tags | Configuration for CloudTrail alarms and tags. | `object` | Review next section for default value | yes |
```
Comment (ipmb) [inline README.md]: We can just call this `tags` and update the description.

File: README.md (hunk 1 above)
Comment (dtorresf) [reply README.md]: Yes typo, I'll fix it

File: README.md [hunk 4]
```diff
@@ -12,114 +12,50 @@ This module creates an AWS CloudTrail trail and the necessary AWS resources to s
...
-      "CIS-3.13-RouteTableChanges" = {
-        description = "3.13 - Ensure a log metric filter and alarm exist for route table changes"
-        pattern     = "{($.eventName=CreateRoute) || ($.eventName=CreateRouteTable) || ($.eventName=ReplaceRoute) || ($.eventName=ReplaceRouteTableAssociation) || ($.eventName=DeleteRouteTable) || ($.eventName=DeleteRoute) || ($.eventName=DisassociateRouteTable)}"
-      }
-      "CIS-3.14-VPCChanges" = {
-        description = "3.14 – Ensure a log metric filter and alarm exist for VPC changes"
-        pattern     = "{($.eventName=CreateVpc) || ($.eventName=DeleteVpc) || ($.eventName=ModifyVpcAttribute) || ($.eventName=AcceptVpcPeeringConnection) || ($.eventName=CreateVpcPeeringConnection) || ($.eventName=DeleteVpcPeeringConnection) || ($.eventName=RejectVpcPeeringConnection) || ($.eventName=AttachClassicLinkVpc) || ($.eventName=DetachClassicLinkVpc) || ($.eventName=DisableVpcClassicLink) || ($.eventName=EnableVpcClassicLink)}"
-      }
-    }
+  cloudtrail_tags = {
```
Comment (ipmb) [inline README.md:43]: this should just be `tags` now, right?

File: README.md (hunk 2 above)
Comment (dtorresf) [reply README.md]: @ipmb done

File: README.md (hunk 3 above)
Comment (dtorresf) [reply README.md]: Yes I updated this @ipmb

File: README.md (hunk 4 above)
Comment (dtorresf) [reply README.md:43]: @ipmb yes I updated this

**Comment by ipmb** [comment]
looks good 👍

**Pull request merged**